frontend/vue/components/CodeExercise/KernelManager.ts

```diff
@@ -264,7 +265,7 @@ export async function requestBinder () {
           resolve(makeSettings(msg))
           break
         default:
-        // console.log(msg);
+          console.log(msg);
```

Comment:
did you mean to leave console logging on?

Reply:
Just forgot to remove it